src/tools/tests/schema2wasm-test.ts

```diff
@@ -21,6 +21,21 @@ import {ParticleSpec} from '../../runtime/particle-spec.js';
 class Schema2Mock extends Schema2Base {
   res: Dictionary<{count: number, adds: string[]}> = {};
 
+  generateEntityClassName(node: SchemaNode, i: number) {
```

Comment:
Same comment here

Reply:
Put the default implementation inside schema2base.